src/backend/storeManagers/nile/library.ts

```diff
@@ -66,6 +67,11 @@ function loadGamesInAccount() {
     } = productDetail
 
     const info = installedGames.get(product.id)
+    // Create save folder name like nile
+    const safeFolderName = ILLEGAL_CHARACTERS.reduce(
+      (acc: string, val: string) => acc.replaceAll(val, ''),
+      title ?? ''
+    )
```

Comment:
Why not use `util.ts` function `removeSpecialcharacters` ?

Reply:
This is what Nile removes from titles, We would need to change what characters are removed by Nile too (that's not an issue at all)